src/Parser.cpp

```diff
@@ -158,20 +155,23 @@ void PacketParser::process_packets(err_code_t& err) {
             mt_kvPtr->cas_unique = 0;
           } else {  // gets
             READ_UNSIGNED(mt_kvPtr->cas_unique);
-            SKIP_BYTES(1); // '\r' after cas
+            SKIP_BYTES(1);  // '\r' after cas
           }
+          m_state = FSM_GET_CAS;
+        }
+        break;
+      case FSM_GET_CAS: // got "cas\r"
```

Comment:
The newly added state `FSM_GET_CAS` should be attached with comment `got "cas\r" or got "bytes\r"`

Reply:
okay, got